backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/pporder/api/impl/HUPPOrderQtyDAO.java

```diff
@@ -161,16 +161,4 @@ public I_PP_Order_Qty retrieveOrderQtyForCostCollector(@NonNull final PPOrderId
 				})
 				.orElse(null);
 	}
-
-	@Override
-	@Cached(cacheName = I_PP_Order_Qty.Table_Name + "#by#" + I_PP_Order_Qty.COLUMNNAME_M_HU_ID)
-	public boolean isHuIdIssued(@NonNull final HuId huId)
```

Comment:
instead of removing it, we shall introduce PP_Order_Qty.Type={I=Issue, R=Receipt}.
`isHuIdIssued` shall also filter by Type=I.

To set the new field, check
de.metas.handlingunits.pporder.api.impl.HUPPOrderQtyDAO.save(CreateIssueCandidateRequest)
de.metas.handlingunits.pporder.api.impl.HUPPOrderQtyDAO.save(CreateReceiptCandidateRequest)

Also, don't forget to migrate the new column!

Reply:
Q: why add 2 different ways of checking if a HU is issued or not? 
This seems silly to me, when we already have the HU.Status field.

Reply:
Confirmed with teo on skype: It's ok to use HU.Status instead of using `PP_Order_Qty`.